### DISCUSSION TITLE
付款后未收到验证码

### DISCUSSION THREAD

**Discussion by vincent2934** [getlantern/lantern-forum-cn#586] [General]
本人于2024年6月16日购买了蓝灯专业版服务，但邮箱至今未收到验证码。邮箱为：465367868@qq.com，
付款成功截图
![payment](https://github.com/getlantern/lantern-forum-cn/assets/15882373/a74c450f-671b-4ebe-a462-bf413e8b0be7)

**Comment by vincent2934** [comment]
@Mike777Khoo

**Comment by MaxMKMax** [comment]
@vincent2934 您好. 已经帮您提交问题了. 但负责人要周二才回来. 我会帮您追问的. 不好意思让您久等了. 请您过 2-3 天再查看这个帖子或邮箱. 我们会给您发送激活码. 谢谢您支持蓝灯!

**Reply by vincent2934** [reply]
目前还是没有收到验证码

**Comment by JOJO0915** [comment]
本人于2024年6月12日购买了蓝灯专业版服务，但邮箱至今未收到验证码。也可能是邮箱拦截问题，当时付款时填写的邮箱是18218651363@163.com。能否核实下然后重新发送验证码，可以确认收件的邮箱为：1275323481@qq.com
PS:付款成功截图
![Uploading 微信图片_20240621011417.jpg…]()
![Uploading sp240621_010908.png…]()

**Reply by MaxMKMax** [reply]
@JOJO0915 https://lantern.surveysparrow.com/s/propayment-triage-form/tt-ZxBeJ
您好. 为了保护您的隐私. 请您发支付单据和账号又想到这个链接. 谢谢. 
您发完后请自行删掉个您的个人资料和单据.

**Comment by MaxMKMax** [comment]
@vincent2934 
https://lantern.surveysparrow.com/s/propayment-triage-form/tt-ZxBeJ
您好. 为了保护您的隐私. 请您发支付单据和账号邮箱到这个链接. 谢谢.

**Reply by vincent2934** [reply]
你好，链接失效了，麻烦再发一次可以吗

**Comment by JOJO0915** [comment]
支付单据和账号已发送，但如果是我的账号邮箱不能接收你们的验证码短信怎么办？是否可以更换接收邮件的邮箱地址？




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "getlantern/lantern-forum-cn"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年6月21日(星期五) 凌晨2:24
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [getlantern/lantern-forum-cn] 付款后未收到验证码 (Discussion #586)





 
https://lantern.surveysparrow.com/s/propayment-triage-form/tt-ZxBeJ
 @vincent2934 您好. 为了保护您的隐私. 请您发支付单据和账号又想到这个链接. 谢谢.
 您发完后请自行删掉个您的个人资料和单据.
